spatial-history/index.md

```diff
@@ -30,10 +19,42 @@ It makes sense to put an introductory paragraph here, above the map and buttons.
+
+# Archaeological Evidence
+Although the saga’s are a valuable source in understanding Viking culture and are good place to start if you want to understand the phenomenon of Vinland, they are not enough to empirically prove that Vikings came to North America. For years the archeological evidence unavailable and Vinland was dismissed as a work of fiction, until Helge Ingstad and Anne Stine Ingstad discovered the Norse settlements of L’anse aux Meadows in 1960. Interestingly, the settlements in L’anse aux Meadows do not include any barns or evidence of livestock (which would imply that it intended as a long-term settlement). Instead, it is made up of dwellings and workshops that suggest that it was used as an access point to the St. Lawrence, a stop over in which to take a break and fix your ship before venturing further south.[^7]
+
+L’anse aux Meadows actually corresponds with a location in the sagas, Leifsbúðir–Straumfjorðr, which has led to speculation about other locations. Helluland and Markland in particular are of interest, because their physical descriptions match almost perfectly with Baffin Island and Labrador. Helluland translates to “Land of Stone Slabs” and is described thusly: “great icebergs were over all up the country, but like a plain of flat stones was all from the sea to the mountains.”[^8] Markland, on the other hand, translates to “Forest Land” is described as “flat, and covered with wood, and white sands were far around where they went, and the shore was low.”[^9]
+
+While we wait for further proof about these other locations, we can be comforted by the fact that more and more information is being discovered everyday. In 2015 Sarah Parcak, an archeologist working with National Geographic, led an excavation team to Point Rosee located on the southern tip of Newfoundland. There they found evidence of Viking structures and a hearth used for roasting iron ore.[^10] It has yet to be confirmed, but if Point Rosee proves to be real it only serves as more evidence in support of Vinland.
+
+## Conclusion
+
+Ultimately, the search for Vinland has allowed archaeologists to prove what historians have long believed; there is historical merit to the Viking sagas. Visualizing the close proximity of Greenland makes it easy to imagine that Vikings were able to come to North America, and the discovery of Viking settlements in Newfoundland allows us to envision exactly how far they ventured.
```

Comment:
Maybe broaden "there is historical merit to the Viging sagas" to a broader discussion of the type of historical conclusion that can be drawn from these kinds of texts.